### ISSUE TITLE
Build problem in cluster-mojo - Cant find pom in maven repository

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dordi/tomgibara#1]
```
What steps will reproduce the problem?
1. svn checkout http://tomgibara.googlecode.com/svn/trunk/cluster
2. cd cluster/cluster-mojo/
3. mvn install

What is the expected output? What do you see instead?
C:\dev\GVM\cluster-mojo>mvn install
[INFO] Scanning for projects...
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project com.tomgibara.cluster:cluster-mojo:1.1-SNAPSHOT 
(C:\dev\GVM\cluster-mojo\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM: Failure to find 
com.tomgibara.cluster:cluster-all:pom:1.0 in http://repo1.maven.org/maven2 was 
cached in the local repository, resolution will not be reattempted
 until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 3, column 11 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e 
switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please 
read the following articles:
[ERROR] [Help 1] 
http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] 
http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
C:\dev\GVM\cluster-mojo>


What version of the product are you using? On what operating system?
Current.  Windows XP/64


Please provide any additional information below.


```

Original issue reported on code.google.com by `rgr.t...@gmail.com` on 19 May 2011 at 2:57

**Comment by GoogleCodeExporter** [comment]
```
Apologies, for some reason I didn't get an email notification when you raised 
this issue.
```

Original comment by `tomgibara` on 1 Jul 2011 at 7:23
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Because the mojo is not a submodule of cluster-all, but only inherits from it, 
the release didn't roll over the parent POM version.

A simple workaround is to checkout the cluster-1.0 tag and build that first, or 
just wait until I commit a revised dependency.
```

Original comment by `tomgibara` on 1 Jul 2011 at 7:28
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed by making cluster-mojo 1.1-SNAPSHOT depend on cluster-all 1.1-SNAPSHOT.
```

Original comment by `tomgibara` on 1 Jul 2011 at 7:34
- Changed state: **Verified**
